### ISSUE TITLE
Warning on createIconSetFromIcoMoon

### ISSUE THREAD

**Issue by brechtwaeghe** [expo/vector-icons#32]
When updating to the latest Expo version, I got this error:

`Expo.Font.style() is deprecated and will be removed in SDK 24. After loading a font with a specified name with Font.loadAsync, you can just reference that font in the fontFamily of your component's style.
- node_modules/expo/src/Font.js:133:2 in style
- node_modules/@expo/vector-icons/createIconSetFromIcoMoon.js:5:53 in default`

Do I have to change something?

**Comment by zorobabel** [comment]
Me too:

![vxo](https://user-images.githubusercontent.com/12666007/33311718-b5b686c0-d3f3-11e7-8566-2a9038fe0f69.png)

I followed these [instructions](https://blog.expo.io/expo-sdk-v23-0-0-is-now-available-be0a8c655414).

**Comment by MrGurns** [comment]
I am also getting this issue. It looks like it was the result of this commit: https://github.com/expo/expo-sdk/commit/3749c6febfda874e712c6d24bc9cacd341031d5d#diff-4e0770d3b45cb15f374541e6d87085f4
Not sure if there is a fix for it or not yet.

**Comment by geovannikun** [comment]
I have this issue with 'MaterialCommunityIcons'.

**Comment by danikane** [comment]
I have the same issue with createIconSetFromFontello.

@MrGurns you are right - it uses FontStyle() as well.
```
import { Font } from 'expo';
import createIconSetFromFontello from './vendor/react-native-vector-icons/lib/create-icon-set-from-fontello';

export default function(config, expoFontName, expoAssetId) {
  return createIconSetFromFontello(config, Font.style(expoFontName).fontFamily, expoAssetId);
}

```

**Comment by brentvatne** [comment]
thanks for pointing this out @danikane - I published 6.2.1 to fix the issue

**Comment by kylev** [comment]
Warning to those with existing apps: SDK 25 has a dependency on `@expo/vector-icons` set to `^6.2.0`. Unfortunately, the last reference to Font.style was removed in 6.3.0.

If you're using `yarn` or another method of locking versions, make sure you upgrade the dependent package (e.g. `yarn upgrade @expo/vector-icons`).

This should probably be added to the upgrade notes, or a 25.0.1 SDK should be issued with a corrected `package.json`. There are a lot of people doing a lot of voodoo that doesn't quite work (according to Google results).
